pkgs/development/python-modules/hid/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, hidapi
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "hid";
+  version = "1.0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-9hsDgvN6M0vIuoYEvIS5SHXuT1lPu6+CssOz6CeIP8E=";
+  };
+
+  propagatedBuildInputs = [ hidapi ];
+
+  checkInputs = [ nose ];
+
+ postPatch = lib.optionalString stdenv.isLinux ''
```

Comment:
Does the package even work on other platforms?

Reply:
I can't know for sure.
The code looks for DLLs, DYLIBs and SOs by searching them directly by filename.

Reply:
I just tested and the patch seems to work on MacOS as well, so we can probably remove the conditional.